src/Aspire.Cli/Program.cs

```diff
@@ -5,24 +5,38 @@
 using System.CommandLine.Parsing;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 
 namespace Aspire.Cli;
 
 public class Program
 {
-    private static IHost BuildApplication()
+    private static IHost BuildApplication(ParseResult parseResult)
     {
         var builder = Host.CreateApplicationBuilder();
+
+        var debugOption = parseResult.GetValue<bool>("--debug");
```

Comment:
We should add another option that waits for the debugger to be attached. Super useful in CLI tools.